docs/manage/_get_support.rst

```diff
@@ -107,3 +64,30 @@ Kolibri needs read and write access to USB drives in order to import and export
 * **Data failures**: Copying the data can take a long time. If you do not see the final success confirmation message after the copy apparently finishes, do not assume that the data has been imported or exported correctly. Restart the process instead, otherwise you risk inconsistent and malfunctioning content data.
 
 * **Hardware life expectancy**: SD and flash storage drives can "expire". Reading and writing large quantities of content data, especially on older or models with smaller capacity, may produce data errors over time.
+
+
+Locate Kolibri log files
+------------------------
+
+When you report a problem with Kolibri, we may ask you to send us Kolibri **log** files to help us find out why is it not working or crashing. 
+
+Open the ``.kolibri`` folder inside the :ref:`Home <home>` folder of the device where Kolibri is running and locate these two files:
+
+* ``kolibri.log``
+* ``debug.log``
+
+.. warning:: On Linux and MacOS systems you will need to activate the *Show Hidden Files* option in your file browser, in order to view the ``.kolibri`` folder.
+
+
+.. _ips:
+
+About IP addresses
+------------------
+
+.. ``0.0.0.0`` = A special IP address on the **server** (your device running Kolibri and "serving" its content to others in the local network), which actually means "all available IP addresses". It's a kind of alias. But accessing ``0.0.0.0`` from another computer doesn't make sense and doesn't work. By default, Kolibri will serve on ``0.0.0.0``, which essentially means all IP addresses that are available on the device will render Kolibri accessible.
```

Comment:
@indirectlylit will probably be happy, but @benjaoming will not, as I went at this with the heavy ax, to the point that it hardly even resembles the original. 

I decided to remove the first item completely, mainly because I don't recall that specific detail about networking ever being needed in context of what the user must understand to make KA Lite or Kolibri work in their deployment.

Reply:
[link to my previous comment](https://github.com/learningequality/kolibri-docs/pull/54#discussion_r230613622)

Reply:
I recall that `0.0.0.0` was printed in some cases where `ifcfg` couldn't detect an address, but it isn't anymore. So it's fine for me to remove this :+1: